### ISSUE TITLE
Product page: Express Payment buttons appear between quantity selector and add to cart button

### ISSUE THREAD

**Issue by nielslange** [woothemes/woocommerce#38737]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Currently, on the product page, the Express Payment buttons appear between the quantity selector and the add to cart button. This issue had been reported by @joaodante in p1686762468600569-slack-C8X6Q7XQU. 

This problem appears both when using a block theme (e.g. TT3) and when using a classic theme (e.g. Storefront).

<table>
<tr>
<td valign="top">WordPress.org site with Twenty Twenty-Three and WooCommerce Stripe Gateway only:
<br><br>

<img width="1080" alt="Screenshot 2023-06-15 at 12 03 33" src="https://github.com/woocommerce/woocommerce/assets/3323310/290a876c-8b4a-4c50-980e-b9d910352ab1">
</td>
<td valign="top">WordPress.com site with Twenty Twenty-Three, WooCommerce Stripe Gateway and WooCommerce Payments:
<br><br>

<img width="1185" alt="Screenshot 2023-06-15 at 12 03 44" src="https://github.com/woocommerce/woocommerce/assets/3323310/0f2e134a-cd56-41e7-81b1-3f71367f32cd">
</td>
</tr>
<tr>
<td valign="top">WordPress.org site with Storefront and WooCommerce Stripe Gateway only:
<br><br>

<img width="1028" alt="Screenshot 2023-06-15 at 12 22 15" src="https://github.com/woocommerce/woocommerce/assets/3323310/b278ed77-1f65-45ad-b296-bcd5cc0ed98c">
</td>
<td valign="top">WordPress.com site with Storefront, WooCommerce Stripe Gateway and WooCommerce Payments:
<br><br>

<img width="1118" alt="Screenshot 2023-06-15 at 12 21 29" src="https://github.com/woocommerce/woocommerce/assets/3323310/7596b711-7c91-40a2-b350-a9c4d562cb20">
</td>
</tr>
</table>

> **Note**
> When using Wordpress.com and having multiple Express Payment buttons from various extensions, the `OR` separator appears below instead of between the Express Payment buttons.

cc: @elizaan36, @shaunandrews

### Expected behavior

- The Express Payment buttons should not appear between the quantity selector and the add to cart button.
- The `OR` separator should be visible between and below the Express Payment buttons.

### Actual behavior

- The Express Payment buttons appear between the quantity selector and the add to cart button.
- All `OR` separators are visible below the Express Payment buttons.

### Steps to reproduce

1. Install, activate and configure WooCommerce Stripe Gateway.
2. Go to `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=methods` and activate `Express checkouts`.
3. Open a product page.
4. Verify that the Express Payment button is placed between the quantity selector and the add to cart button.

### WordPress Environment

<details>
  <summary>SSR of WordPress.org site</summary>
  
  ```
### WordPress Environment ###

WordPress address (URL): https://total-skunk.jurassic.ninja
Site address (URL): https://total-skunk.jurassic.ninja
WC Version: 7.8.0
REST API Version: ✔ 7.8.0
WC Blocks Version: ✔ 10.2.3
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.2.2
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache/2.4.57 (Unix) OpenSSL/1.0.2g
PHP Version: 7.4.33
PHP Post Max Size: 1 GB
PHP Time Limit: 30
PHP Max Input Vars: 5000
cURL Version: 7.47.0
OpenSSL/1.0.2g

SUHOSIN Installed: –
MySQL Version: 5.7.33-0ubuntu0.16.04.1-log
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.8.0
WC Database Prefix: wp_
Total Database Size: 6.13MB
Database Data Size: 4.60MB
Database Index Size: 1.53MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 3.48MB + Index: 0.06MB + Engine InnoDB
wp_postmeta: Data: 0.13MB + Index: 0.09MB + Engine InnoDB
wp_posts: Data: 0.06MB + Index: 0.06MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 22
page: 9
post: 16
product: 18
product_variation: 7
shop_coupon: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ❌Error messages should not be shown to visitors.

### Active Plugins (3) ###

Companion Plugin: by Osk – 1.29
WooCommerce Stripe Gateway: by WooCommerce – 7.4.1
WooCommerce: by Automattic – 7.8.0

### Inactive Plugins (4) ###

Jetpack: by Automattic – 12.2.1
WooCommerce Payments: by Automattic – 6.0.0
WooCommerce Payments Dev Tools: by Automattic –
WordPress Importer: by wordpressdotorg – 0.8.1

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: ✔
HPOS feature screen enabled: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: –

### WC Pages ###

Shop base: #54 - /shop/classic-shop/
Cart: #55 - /cart/
Checkout: #56 - /checkout/
My account: #59 - /my-account/
Terms and conditions: #60 - /terms/

### Theme ###

Name: Twenty Twenty-Three
Version: 1.1
Author URL: https://wordpress.org
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: /srv/users/user49bd1fe6/apps/user49bd1fe6/public/wp-content/plugins/woocommerce/packages/woocommerce-blocks/templates/notices/error.php
/srv/users/user49bd1fe6/apps/user49bd1fe6/public/wp-content/plugins/woocommerce/packages/woocommerce-blocks/templates/notices/notice.php
/srv/users/user49bd1fe6/apps/user49bd1fe6/public/wp-content/plugins/woocommerce/packages/woocommerce-blocks/templates/notices/success.php


### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: core-profiler
minified-js
new-product-management-experience
product-variation-management
settings
async-product-editor-category-field

Daily Cron: ✔ Next scheduled: 2023-06-16 08:16:48 +00:00
Options: ✔
Notes: 57
Onboarding: -

### Action Scheduler ###

Complete: 5
Oldest: 2023-06-15 08:17:10 +0000
Newest: 2023-06-15 09:56:22 +0000

Pending: 2
Oldest: 2023-06-15 10:16:53 +0000
Newest: 2023-06-16 08:17:10 +0000

Failed: 1
Oldest: 2023-06-15 09:58:23 +0000
Newest: 2023-06-15 09:58:23 +0000


### Status report information ###

Generated at: 2023-06-15 10:16:17 +00:00
```
</details>

<details>
  <summary>SSR of WordPress.org site</summary>
  
```
### WordPress Environment ###

WordPress address (URL): https://atomic.nielslange.blog
Site address (URL): https://atomic.nielslange.blog
WC Version: 7.8.0
REST API Version: ✔ 7.8.0
WC Blocks Version: ✔ 10.2.3
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.2.2
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 8.0.29
PHP Post Max Size: 2 GB
PHP Time Limit: 1200
PHP Max Input Vars: 6144
cURL Version: 8.0.1
OpenSSL/1.1.1n

SUHOSIN Installed: –
MySQL Version: 10.4.26-MariaDB-log
Max Upload Size: 2 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.8.0
WC Database Prefix: wp_
Total Database Size: 14.25MB
Database Data Size: 12.68MB
Database Index Size: 1.57MB
wp_woocommerce_sessions: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.11MB + Index: 0.13MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.08MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_options: Data: 2.11MB + Index: 0.06MB + Engine InnoDB
wp_postmeta: Data: 6.02MB + Index: 0.09MB + Engine InnoDB
wp_posts: Data: 3.13MB + Index: 0.06MB + Engine InnoDB
wp_taxjar_record_queue: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.11MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.08MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.09MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

amp_validated_url: 7
attachment: 24
customize_changeset: 1
jetpack_migration: 2
jp_img_sitemap: 9
jp_sitemap: 9
jp_sitemap_master: 9
nav_menu_item: 5
page: 7
post: 1
product: 18
product_variation: 7
revision: 3
shop_order: 1
wp_global_styles: 2
wp_navigation: 1
wp_template_part: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (6) ###

Akismet Anti-Spam: by Automattic – 5.1
Jetpack: by Automattic – 12.3-a.5
WooCommerce Stripe Gateway: by WooCommerce – 7.4.1
WooCommerce Payments Dev Tools: by Automattic –
WooCommerce Payments: by Automattic – 6.0.0
WooCommerce: by Automattic – 7.8.0

### Inactive Plugins (43) ###

AMP: by AMP Project Contributors – 2.4.1
AutomateWoo: by WooCommerce – 5.7.7
Avalara AvaTax: by Avalara – 2.5.0
Classic Editor: by WordPress Contributors – 1.6.3
CoBlocks: by GoDaddy – 3.0.3
Crowdsignal Forms: by Automattic – 1.7.0
Crowdsignal Polls & Ratings: by Automattic
Inc. – 3.0.10

Facebook for WooCommerce: by Facebook – 3.0.25
Google Listings and Ads: by WooCommerce – 2.4.10
Gutenberg: by Gutenberg Team – 15.9.1
Klarna Checkout for WooCommerce: by Krokedil – 2.10.2
Klarna Payments for WooCommerce: by krokedil
klarna
automattic – 3.0.7

Layout Grid: by Automattic – 1.8.3
Mailchimp for WooCommerce: by Mailchimp – 2.8.3
Page Optimize: by Automattic – 0.5.3
Pinterest for WooCommerce: by WooCommerce – 1.3.3
Storefront Powerpack: by WooCommerce – 2.0.0
TaxJar - Sales Tax Automation for WooCommerce: by TaxJar – 4.1.5
TikTok: by TikTok – 1.1.1
WooCommerce Australia Post Shipping: by WooCommerce – 2.5.1
WooCommerce Back In Stock Notifications: by WooCommerce – 1.6.2
WooCommerce Blocks: by Automattic – 10.4.2
WooCommerce Brands: by WooCommerce – 1.6.52
WooCommerce Canada Post Shipping: by WooCommerce – 2.8.1
WooCommerce EU VAT Number: by WooCommerce – 2.8.4
WooCommerce Eway Payment Gateway: by WooCommerce – 3.6.1
WooCommerce FedEx Shipping: by WooCommerce – 3.7.3
WooCommerce Gift Cards: by WooCommerce – 1.16.2
WooCommerce Google Analytics Integration: by WooCommerce – 1.8.2
WooCommerce Min/Max Quantities: by WooCommerce – 4.1.0
WooCommerce Payfast Gateway: by WooCommerce – 1.5.4
WooCommerce PayPal Checkout Gateway: by WooCommerce – 2.1.3
WooCommerce Product Add-ons: by WooCommerce – 6.4.0
WooCommerce Product Bundles: by WooCommerce – 6.20.0
WooCommerce Product Recommendations: by WooCommerce – 2.3.0
WooCommerce Royal Mail: by WooCommerce – 2.8.1
WooCommerce Shipment Tracking: by WooCommerce – 2.3.1
WooCommerce Shipping & Tax: by WooCommerce – 2.2.5
WooCommerce Square: by WooCommerce – 3.8.3
WooCommerce UPS Shipping: by WooCommerce – 3.4.3
WooCommerce USPS Shipping: by WooCommerce – 4.6.3
WordPress.com Editing Toolkit: by Automattic – 3.68337
WP Rollback: by GiveWP.com – 1.7.3

### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
object-cache.php: Memcached

### Must Use Plugins (1) ###

WP.com Site Helper: by  –

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: –
HPOS feature screen enabled: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: –

### WC Pages ###

Shop base: #5 - /
Cart: #6 - /cart/
Checkout: #7 - /checkout/
My account: #8 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty-Three
Version: 1.1
Author URL: https://wordpress.org
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: /wordpress/plugins/woocommerce/7.8.0/packages/woocommerce-blocks/templates/notices/error.php
/wordpress/plugins/woocommerce/7.8.0/packages/woocommerce-blocks/templates/notices/notice.php
/wordpress/plugins/woocommerce/7.8.0/packages/woocommerce-blocks/templates/notices/success.php


### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live
Subscriptions Live URL: https://atomic.nielslange.blog
Subscriptions-core Library Version: 5.7.2
Subscription Statuses: –
WooCommerce Account Connected: ✔ Yes
Active Product Key: ❌ No

### Store Setup ###

Country / State: United States (US) — California

### Subscriptions by Payment Gateway ###


### Payment Gateway Support ###

WooCommerce Payments: products
refunds
multiple_subscriptions
subscription_cancellation
subscription_payment_method_change_admin
subscription_payment_method_change_customer
subscription_payment_method_change
subscription_reactivation
subscription_suspension
subscriptions
gateway_scheduled_payments
tokenization
add_payment_method

Stripe: products
refunds
tokenization
add_payment_method


### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: core-profiler
minified-js
navigation
new-product-management-experience
product-variation-management
settings
async-product-editor-category-field

Daily Cron: ✔ Next scheduled: 2023-06-16 08:26:39 +00:00
Options: ✔
Notes: 152
Onboarding: completed

### WooCommerce Payments ###

Version: 6.0.0
Connected to WPCOM: Yes
Blog ID: 155632866
Account ID: acct_1MfxFzCEZ5omP7EF

### Action Scheduler ###

Complete: 44
Oldest: 2023-05-15 18:55:12 +0000
Newest: 2023-06-15 09:46:31 +0000

Pending: 2
Oldest: 2023-06-15 10:19:10 +0000
Newest: 2023-06-15 21:36:36 +0000

Failed: 19
Oldest: 2022-11-03 00:59:52 +0000
Newest: 2023-06-01 18:02:59 +0000


### Status report information ###

Generated at: 2023-06-15 10:18:49 +00:00
```
</details>

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by elizaan36** [comment]
Thanks for the ping @nielslange 

This is also being discussed [here](https://github.com/Automattic/woopay/issues/1854#issuecomment-1593550901). 

The ideal solution for Product and Cart pages would be to:
- Remove the -OR- separator completely
- Move "Add to Cart" or "Proceed to checkout" to the first position
- Make the primary action full width to align with the express buttons

<img width="712" alt="image" src="https://github.com/woocommerce/woocommerce/assets/5121465/d66eb6f1-85eb-417f-8534-4e2ef051199c">

<img width="704" alt="image" src="https://github.com/woocommerce/woocommerce/assets/5121465/a5b4d447-2c1d-425b-ab73-ce4d429722ce">

**Comment by pierorocca** [comment]
Closing as this was resolved in #1854.
